sdk/eventhub/azure-eventhubs/azure/eventhub/_common.py

```diff
@@ -378,28 +321,27 @@ def try_add(self, event_data):
         :raise: :class:`ValueError`, when exceeding the size limit.
         """
         if event_data is None:
-            log.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
+            _LOGGER.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
             return
         if not isinstance(event_data, EventData):
-            raise TypeError('event_data should be type of EventData')
+            raise TypeError('event_data should be an EventData instance.')
 
         if self._partition_key:
             if event_data.partition_key and event_data.partition_key != self._partition_key:
-                raise EventDataError('The partition_key of event_data does not match the one of the EventDataBatch')
+                raise EventDataError('The partition key of event_data does not match the partition key of this batch.')
```

Comment:
This is a `ValueError`. correct?

Reply:
This should just be a ValueError anyway, like it is just a few lines down....